src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheck.java

```diff
@@ -73,6 +73,11 @@ public void setOffset(int offset) {
     }
 
     @Override
+    public int[] getRequiredJavadocTokens() {
+        return getDefaultJavadocTokens();
```

Comment:
there should be getAcceptableJavadocTokens usage. We require all. 
Please fix in all other cases.